### ISSUE TITLE
Nested display problems

### ISSUE THREAD

**Issue by shuojie819** [mugiwara85/CodeblockCustomizer#12]
![image](https://github.com/mugiwara85/CodeblockCustomizer/assets/72399630/278cf96e-234f-4875-8859-807f398c1078)

Sample text
`````md
````ad-flex
```ad-blank
>个人记录
- ✔ 待办事项
- ✅ 已完成事项
- ▶ 观影记录
- 📙 阅读记录
- ...

```

```ad-blank
>工作记录
- 🚩工作计划
- 🏳 未完成的计划
- ☢ Bug清单
- 🔀 API接口文档
- ...

```

```ad-blank
>教育记录
- ♎课程管理
- 📝 写作计划
- ✏ 实验记录
- ...

```
````
`````

**Comment by shuojie819** [comment]
Ok, I know that three backquotes are the correct syntax, I used the Admonition plugin, which uses the syntax of code block nesting, Use 4 or more back quotes, That invalidated the exclusion language I configured.

Sorry, my English is not good
It's not a big question, it's just that I suddenly found it and brought it up.

**Comment by mugiwara85** [comment]
Don't worry about your English. It is not my native language either :)

If you want to exclude the first and the last line you could add following to your exclusion:
ad-*, `ad-*, `,
The first (ad-*) excludes every code block where the defined language starts with ad-. The second (`ad-*) does almost the same but since you wrote 4 backticks, the 4th counts as the language that's why you have to write a backtick at the beginning. Similar to the last (`). You wrote also 4 backticks. This would be the closing for the outer code block, so the 4th backtick counts as the defined language for the code block. After that it should look like this:
![image](https://github.com/mugiwara85/CodeblockCustomizer/assets/65603765/338f970f-8cb3-485e-93fd-7a574da36c56)

Btw, as far as I know Admonitions are obsolete. You can use callouts now: https://help.obsidian.md/Editing+and+formatting/Callouts

Nesting a code block and a callout/admonition is somewhat complex. I am looking for a solution, but unfortunately it is not that easy so it can take some time.

**Comment by shuojie819** [comment]
Okay, I am currently using both grammars.
Nested display is not a major issue and does not require immediate repair.

**Comment by mugiwara85** [comment]
Sorry, it took this long, but it is fixed in release 1.2.1. It should work now correctly.
